src/lib/constants.js

```diff
@@ -11,9 +13,15 @@ const WEEKEND = [
   /week\-?end/i,
 ]
 
+
+// other patterns
+const USER_ALREADY_INTRODUCED = /suis|am|\'m/i
```

Comment:
That is the plan that I put in the _Project_ tab in this repo.
It's either via a direct message or I private conversation, just like the message _SlackBot_ echos; they appear to you and only you, wherever you are.